schemas/view-schema.json

```diff
@@ -219,6 +219,64 @@
                 "readOnly": true
               },
               "readOnly": true
+            },
+            "metric": {
```

Comment:
Existing metrics are directly in `view.*`. While I agree that it could be nice to group metrics in `view.metrics.*`, we lose a bit of consistency. Not a blocker IMO, but I'm curious of other opinions.

Reply:
agree, I would even go a bit further and remove the other grouping level:

```
view.memory_average
view.cpu_ticks_per_second
```

Reply:
Also wouldn't this be more consistent with other values (we are using `view.error.count` and not `view.error_count`)

Reply:
since v2, we only have nested attributes for core concepts and try to inline everything else:
```
// group for error
view.error.count

// no group for country
geo.country_iso_code
```

in the UI, we could still have:
- dedicated facet groups
- dedicated section in view side panel